src/nvim/ex_docmd.c

```diff
@@ -8761,7 +8761,10 @@ makeopens (
      * Do this before restoring the view, so that the topline and the
      * cursor can be set.  This is done again below.
      */
-    if (put_line(fd, "set winheight=1 winwidth=1") == FAIL)
+    if (put_line(
+          fd,
+          "set winminheight=1 winminwidth=1 winheight=1 winwidth=1"
+        ) == FAIL)
```

Comment:
Linter should be fine with `fd, "…"` on its own line, but it expects `)` to be with `if` then AFAIR.